packages/web/docs/src/components/team-section.tsx

```diff
@@ -1,3 +1,4 @@
+import { Fragment } from 'react';
```

Comment:
![CleanShot 2024-08-15 at 08 51 44@2x](https://github.com/user-attachments/assets/ffbb99e9-f54c-4fdc-bad2-30b65f8130c8)

Reply:
I have to get an autofix for this.